src/Symfony/Component/Console/Terminal.php

```diff
@@ -106,17 +106,17 @@ private static function getConsoleMode()
                 return [(int) $matches[2], (int) $matches[1]];
             }
         }
+
+        return null;
     }
 
     /**
      * Runs and parses stty -a if it's available, suppressing any error output.
-     *
-     * @return string|null
      */
-    private static function getSttyColumns()
+    private static function getSttyColumns(): ?string
     {
         if (!\function_exists('proc_open')) {
-            return;
+            return null;
```

Comment:
After adding the return type, php will error if I don't explicitly return `null`.

Reply:
ref #17201